about.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles/global.css" />
+    <link rel="stylesheet" href="./styles/header.css" />
+    <link rel="stylesheet" href="./styles/main.css" />
+    <link rel="stylesheet" href="./styles/about.main.css" />
+    <link rel="stylesheet" href="./styles/footer.css" />
+    <script src="./scripts/header.js" type="module"></script>
+    <title>About</title>
+</head>
+<body>
+    <header>
+        <div class="humberger">
+            <div class="bar"></div>
+        </div>
+    
+        <div class="navs">
+            <nav class="nav-1">
+                <ul>
+                    <li>
+                        <a class="nav-link" href="#">
+                            <img src="./media/facebook.png" alt="facebook" />
+                        </a>
+                    </li>
+                    <li>
+                        <a class="nav-link" href="#">
+                            <img src="./media/twitter.png" alt="twitter" />
+                        </a>
+                    </li>
+    
+                    <li><a class="nav-link" href="#">English</a></li>
+                    <li><a class="nav-link" href="#">My Page</a></li>
+                    <li><a class="nav-link" href="#">Logout</a></li>
+                </ul>
+            </nav>
+            <div class="nav-2-container">
+                <div class="logo">
+                    <a href="./index.html">
+                    <img src="./media/logo_64.png" alt="top totutials logo" />
+                    </a>
+                </div>
+                <nav class="nav-2">
+                    <ul>
+                        <li>
+                            <a class="nav-link" href="./about.html">About</a>
+                        </li>
+                        <li>
+                            <a class="nav-link" href="./index.html#program">Program</a>
+                        </li>
+                        <li>
+                            <a class="nav-link" href="./index.html#join">Join</a>
+                        </li>
+                        <li>
+                            <a class="nav-link" href="./index.html#sponsor">Sponsor</a>
+                        </li>
+                        <li class="get-started">
+                            <a class="nav-link" href="./index.html#">Get Started</a>
+                        </li>
+                    </ul>
+                </nav>
```

Comment:
- [Optional  ] For User interaction I recommend you to add the link to "Home" page from "About" page at the mobile menu for mobile screen size .

![Capture](https://user-images.githubusercontent.com/93598189/196501130-677499a4-e305-4d9c-8cea-59193f6e84b1.PNG)